looker/provider_test.go

```diff
@@ -34,3 +73,51 @@ func newTestLookerSDK() (*client.LookerSDK, error) {
 func TestMain(m *testing.M) {
 	resource.TestMain(m)
 }
+
+func filterCredentials(i *cassette.Interaction) error {
+	if strings.Contains(i.Request.Body, "client_id") || strings.Contains(i.Request.Body, "client_secret") {
+		form := make(url.Values)
+		form.Add("client_id", "[REDACTED]")
+		form.Add("client_secret", "[REDACTED]")
+
+		i.Request.Form = form
+		i.Response.Body = `{"access_token": "[REDACTED]"}`
+	}
```

Comment:
Could we do something like:
```suggestion
	_, ok := i.Request.Form["client_id"]
	if ok {
		i.Request.Form.Set("client_id", "[REDACTED]")
	}

	_, ok = i.Request.Form["client_secret"]
	if ok {
		i.Request.Form.Set("client_id", "[REDACTED]")
	}
```
Should we attempt to parse the JSON to redact certain keys like we are doing here with the body because the user API test is also leaking credentials?

Reply:
I've tried that in the latest commit, but it gets messy since the body isn't always a marshallble JSON, even worse the errors aren't logged nor do they stop execution so it took me a while to figure out why it wasn't working.